Sources/SwiftyTailwind/ChecksumValidation.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  File.swift
+//  
+//
+//  Created by Brady Klein on 5/10/24.
+//
+
+import Foundation
+import CommonCrypto
+import TSCBasic
+
+protocol ChecksumValidating {
+    func generateChecksumFrom(_ filePath: AbsolutePath) throws -> String
+    func compareChecksum(from filePath: AbsolutePath, to checksum: String) throws -> Bool
+}
+
+struct ChecksumValidation: ChecksumValidating {
+    func generateChecksumFrom(_ filePath: AbsolutePath) throws -> String {
+        let binaryData = try Data(contentsOf: filePath.asURL)
```

Comment:
This will load the content of the binary into memory, which is not ideal. What if we use a system process to run the following command?
```
shasum -a 256 /path/to/your/file
```

Reply:
Good shout! I've updated to use a system process instead of hashing locally in memory.